charts/postgres-operator/values.yaml

```
@@ -23,6 +23,8 @@ configGeneral:
   enable_shm_volume: "true"
   # etcd connection string for Patroni. Empty uses K8s-native DCS.
   etcd_host: ""
+  # Select if setup uses endpoints (default), or configmaps to manage leader (DCS=k8s)
+  # kubernetes_use_configmaps: false
```

```suggestion
  # kubernetes_use_configmaps: "false"
```
For configmap we have to quote the bools.